autoscheduler/frontend/src/components/Schedule/Schedule.css

```diff
@@ -74,11 +76,12 @@
     flex-grow: 1;
     display: flex;
     flex-direction: column;
-    z-index: -1;
-    background-color: white;
+    background: linear-gradient(transparent 0, transparent 0.5rem, white 0.5rem);
```

Comment:
You aren’t missing anything, as there is not actually any real gradient. Instead, I’m using linear gradient to make the top 0.5rem of the calendar day transparent so that it doesn’t  cover up the header.